raiwidgets/tests/test_cohort.py

```diff
@@ -497,40 +497,46 @@ def test_cohort_serialization_single_value_method(self, method):
                                        arg=[65], column='age')
         cohort_1 = Cohort(name="Cohort New")
         cohort_1.add_cohort_filter(cohort_filter_1)
-        json_str = json.dumps(cohort_1,
-                              default=cohort_filter_json_converter)
+        json_str = cohort_1.to_json()
 
         assert 'Cohort New' in json_str
         assert method in json_str
         assert '[65]' in json_str
         assert 'age' in json_str
 
-    def test_cohort_serialization_in_range_method(self):
+    def test_cohort_serialization_deserialization_in_range_method(self):
```

Comment:
Do we need tests for validation exception scenarios that are listed above in cohort.py?

Reply:
added